credsweeper/rules/config.yaml

```diff
@@ -229,15 +229,25 @@
   severity: high
   type: multi
   values:
-    - (?P<value>[0-9]+\-[0-9A-Za-z_]{32}\.apps\.googleusercontent\.com)
-    - (?<![0-9a-zA-Z_-])(?P<value>[0-9a-zA-Z_-]{24})([^=0-9A-Za-z_/+-]|$)
+    - (?P<value>[0-9]{3,}-[0-9a-z_]{32}\.apps\.googleusercontent\.com)
```

Comment:
This regexp is vulnerable to regexp bomb input. Checked with https://devina.io/redos-checker

Reply:
Thanks for pointed the issue. 
It can increase time of scan but not failure appear.
Huge lines (over 8000 symbols) are skipped before regex and the attack cannot be sucessful.

Reply:
Well done